### DISCUSSION TITLE
"Double-click" to add link?

### DISCUSSION THREAD

**Discussion by dmb0058** [clientIO/joint#2691] [Q&A] [answered]
### Introduction

I think there might be a simple answer to this - either "Yes, just set xxx", or "No, you need to write code 😄

I have a component where the items are rectangles, each with a port on the left.To connect the items, the user drags a link from the port on the source item and releases it on the target. I use the "link:connect" event to trap the release.

It would be more accessible if this also worked without having to hold the moise button down, i.e. to click and release on the source port, then have the link follow the mouse as it moves, until the target shape is clicked. The behaviour of clicking anywhere else such as on the background, should be the same as releasing a drag, i.e. the link disappears.

Is this behaviour supported by setting some flag (like link pinning for example)?

Thanks for any info,

David

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
Duplicate of https://github.com/clientIO/joint/discussions/2301.
